### PR TITLE
v1.5.0 release

### PR DESCRIPTION
**Pull request by just4brown**
*Issue #, if available:* https://github.com/aws/sagemaker-distribution/milestone/12

*Description of changes:* Release artifacts for new version 1.5.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by just4brown** [comment]
Tests are passing aside from known issues listed below

https://github.com/aws/sagemaker-distribution/issues/30
https://github.com/aws/sagemaker-distribution/issues/110

**Review by trajanikant — APPROVED** [review]

**Comment by just4brown** [comment]
# Staleness Report: 1.5.0(gpu)

Package | Current Version in the Distribution image | Latest Relevant Version in Upstream
---|---|---
numpy|1.26.4|1.26.4
jinja2|3.1.3|3.1.3
${\color{red}pandas}$|2.1.4|2.2.0
altair|5.2.0|5.2.0
${\color{red}boto3}$|1.28.64|1.34.37
ipython|8.21.0|8.21.0
jupyter-lsp|2.2.2|2.2.2
jupyterlab|4.1.0|4.1.0
amazon-codewhisperer-jupyterlab-ext|2.0.1|2.0.1
jupyter-scheduler|2.5.0|2.5.0
amazon-sagemaker-jupyter-scheduler|3.0.6|3.0.6
${\color{red}scipy}$|1.11.4|1.12.0
scikit-learn|1.4.0|1.4.0
pip|23.3.2|23.3.2
${\color{red}torchvision}$|0.15.2|0.16.1
autogluon|0.8.2|0.8.2
ipywidgets|8.1.1|8.1.1
notebook|7.0.7|7.0.7
aws-glue-sessions|1.0.4|1.0.4
conda|23.11.0|23.11.0
${\color{red}fastapi}$|0.103.2|0.109.2
langchain|0.1.5|0.1.5
${\color{red}jupyter-ai}$|2.9.1|2.10.0
jupyter-dash|0.4.2|0.4.2
jupyter-server-proxy|4.1.0|4.1.0
jupyterlab-git|0.50.0|0.50.0
jupyterlab-lsp|5.0.3|5.0.3
${\color{red}keras}$|2.12.0|2.15.0
matplotlib|3.8.2|3.8.2
${\color{red}nodejs}$|18.18.2|18.19.0
py-xgboost-gpu|1.7.6|1.7.6
thrift_sasl|0.4.3|0.4.3
pyhive|0.7.0|0.7.0
python-gssapi|1.8.3|1.8.3
python-lsp-server|1.10.0|1.10.0
${\color{red}pytorch-gpu}$|2.0.0|2.1.2
sagemaker-headless-execution-driver|0.0.12|0.0.12
sagemaker-jupyterlab-emr-extension|0.1.9|0.1.9
sagemaker-jupyterlab-extension|0.2.0|0.2.0
sagemaker-kernel-wrapper|0.0.2|0.0.2
${\color{red}sagemaker-python-sdk}$|2.198.1|2.206.0
sagemaker-studio-analytics-extension|0.0.21|0.0.21
sasl|0.3.1|0.3.1
supervisor|4.2.5|4.2.5
${\color{red}tensorflow}$|2.12.1|2.15.0

# Staleness Report: 1.5.0(cpu)

Package | Current Version in the Distribution image | Latest Relevant Version in Upstream
---|---|---
numpy|1.26.4|1.26.4
jinja2|3.1.3|3.1.3
${\color{red}pytorch}$|2.0.0|2.1.2
${\color{red}pandas}$|2.1.4|2.2.0
altair|5.2.0|5.2.0
${\color{red}boto3}$|1.28.64|1.34.37
ipython|8.21.0|8.21.0
jupyter-lsp|2.2.2|2.2.2
jupyterlab|4.1.0|4.1.0
amazon-codewhisperer-jupyterlab-ext|2.0.1|2.0.1
jupyter-scheduler|2.5.0|2.5.0
amazon-sagemaker-jupyter-scheduler|3.0.6|3.0.6
${\color{red}scipy}$|1.11.4|1.12.0
scikit-learn|1.4.0|1.4.0
pip|23.3.2|23.3.2
${\color{red}torchvision}$|0.15.2|0.16.1
autogluon|0.8.2|0.8.2
ipywidgets|8.1.1|8.1.1
notebook|7.0.7|7.0.7
aws-glue-sessions|1.0.4|1.0.4
conda|23.11.0|23.11.0
${\color{red}fastapi}$|0.103.2|0.109.2
langchain|0.1.5|0.1.5
${\color{red}jupyter-ai}$|2.9.1|2.10.0
jupyter-dash|0.4.2|0.4.2
jupyter-server-proxy|4.1.0|4.1.0
jupyterlab-git|0.50.0|0.50.0
jupyterlab-lsp|5.0.3|5.0.3
${\color{red}keras}$|2.12.0|2.15.0
matplotlib|3.8.2|3.8.2
${\color{red}nodejs}$|18.18.2|18.19.0
py-xgboost-cpu|1.7.6|1.7.6
thrift_sasl|0.4.3|0.4.3
pyhive|0.7.0|0.7.0
python-gssapi|1.8.3|1.8.3
python-lsp-server|1.10.0|1.10.0
sagemaker-headless-execution-driver|0.0.12|0.0.12
sagemaker-jupyterlab-emr-extension|0.1.9|0.1.9
sagemaker-jupyterlab-extension|0.2.0|0.2.0
sagemaker-kernel-wrapper|0.0.2|0.0.2
${\color{red}sagemaker-python-sdk}$|2.198.1|2.206.0
sagemaker-studio-analytics-extension|0.0.21|0.0.21
sasl|0.3.1|0.3.1
supervisor|4.2.5|4.2.5
${\color{red}tensorflow}$|2.12.1|2.15.0
